src/librawspeed/decoders/DngDecoderSlices.cpp

```diff
@@ -418,10 +419,9 @@ void DngDecoderSlices::decodeSlice(DngDecoderThread* t) {
     while (!t->slices.empty()) {
       DngSliceElement e = t->slices.front();
       t->slices.pop();
-      LJpegPlain l(*mFile, e.byteOffset, e.byteCount, mRaw);
-      l.mDNGCompatible = mFixLjpeg;
+      DngDecompressor d(*mFile, e.byteOffset, e.byteCount, mRaw);
```

Comment:
Now that I decided to put the last patch in a separate pr, it would be easy to change the name of the `DngDecompressor`. Would you please comment on my `DngLJpegDecompressor` suggestion before I push that again?